app/javascript/packs/application.js

```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
```

Comment:
file này dùng để làm gì

Reply:
file này sẽ là entry point cho toàn bộ code JavaScript. Tự động load các file js trong ứng dụng
Link tham khaor: https://viblo.asia/p/tim-hieu-ve-webpacker-o-rails-6-part-1-tong-quan-ve-webpacker-va-packs-trong-rails-6-V3m5WP8QKO7